.github/workflows/codeql-analysis.yml

```diff
@@ -14,6 +14,9 @@ name: "CodeQL"
 on:
   push:
     branches: [ master ]
+    # Workflows triggered by Dependabot on the "push" event run with read-only access
+    # Code Scanning requires write access, so ignore dependabot branches
+    branches-ignore: [dependabot/**]
```

Comment:
Huh, so the fact that it's not mentioned in the list isn't enough? My intention with this config was to make it only run on matter periodically and never run on a PR. Do you know where my misunderstanding was?

Reply:
Hmm good point. On a 2nd pass, I found that this is an incorrect solution, and will document why that is on the PR itself.